### DISCUSSION TITLE
Ghost touches since installing TMF rom

### DISCUSSION THREAD

**Discussion by ExtremeOne** [remay/tmf-hudl#36] [Help]
I decided to resurrect an old HUDL that my daughter used to own. Seems a shame to send it to landfill

Anyway my investigations lead me here. :)

I've flashed TMF 1.0 from an SD card and all went well, apart from the fact that I'm getting dozens of ghost touches!

I've enabled Developer mode and by enabling viewing of Screen Touches I'm seeing touches all over the screen, nowhere near where my fingers are.

Any ideas?

**Comment by ExtremeOne** [comment]
Never mind. It was due to a faulty USB lead / USB charger. After switching to a different lead and charger no more ghost touches.

**Reply by remay** [reply]
Glad it wasn't the ROM :-).  IT shouldn't have been as I didn't do that at all.  If you haven't seen it I was going to point you at the touch screen reset under the display settings.

I've seen something similar once before that turned out to be a dodgy battery.

Great that you've got it going.
